doc/src/manual/constructors.md

```diff
@@ -539,11 +549,11 @@ that specific type parameters cannot be requested manually.
 For example, say we define a type that stores a vector along with an accurate representation of
 its sum:
 
-```julia
-type SummedArray{T<:Number,S<:Number}
-    data::Vector{T}
-    sum::S
-end
+```jldoctest
```

Comment:
Considering how many weird syntax errors we've had I don't see the harm in having this as a doctest. If you'd like I can add an example of constructing this type?